app/endpoints/notification.py

```diff
@@ -235,3 +235,22 @@ async def send_notif(
         message=message,
     )
     print("send notif done")
+
+
+@router.get(
```

Comment:
What it is this endpoint for ?

Reply:
This is a debug endpoint allowing admins to know which what are their Firebase tokens. Do we want to keep it or should I remove it?

Reply:
It could be useful for debugging, let's keep it